app/src/main/java/org/simple/clinic/bp/entry/BloodPressureEntryViewRenderer.kt

```diff
@@ -3,12 +3,15 @@ package org.simple.clinic.bp.entry
 import org.simple.clinic.bp.entry.OpenAs.New
 import org.simple.clinic.bp.entry.OpenAs.Update
 import org.simple.clinic.mobius.ViewRenderer
+import org.simple.clinic.util.ValueChangedCallback
 
 class BloodPressureEntryViewRenderer(
     val ui: BloodPressureEntryUi
 ) : ViewRenderer<BloodPressureEntryModel> {
+  private val openAsValueChangedCallback = ValueChangedCallback<OpenAs>()
```

Comment:
This is just a workaround for the fact that we only need to set this UI up once. I am assuming this will eventually become an effect that gets emitted from the init function?

Reply:
No, the view renderer is invoked every time the model changes. In order to enable granular updates on the UI, we use this facility.